pkg/actor/validate_version.go

```diff
@@ -125,30 +129,35 @@ func (v *versionChecker) Act(ctx context.Context, cluster *resource.Cluster) err
 		log.Error(err, "failed to reconcile job only err")
 	}
 
-	if changed {
-		log.V(int(zapcore.DebugLevel)).Info("created/updated job, stopping request processing")
-		CancelLoop(ctx)
-		return nil
-	}
-
-	log.V(int(zapcore.DebugLevel)).Info("version checker", "job", jobName)
+	// if changed {
+	// 	log.V(DEBUGLEVEL).Info("created/updated job, stopping request processing")
+	// 	CancelLoop(ctx)
+	// 	return nil
+	// }
+	log.V(DEBUGLEVEL).Info("version checker", "job", jobName)
+	job := &kbatch.Job{}
 	key := kubetypes.NamespacedName{
 		Namespace: cluster.Namespace(),
 		Name:      jobName,
 	}
-	job := &kbatch.Job{}
 
 	clientset, err := kubernetes.NewForConfig(v.config)
 	if err != nil {
 		log.Error(err, "cannot create k8s client")
 		return errors.Wrapf(err, "check version failed to create kubernetes clientset")
 	}
 	if err := v.client.Get(ctx, key, job); err != nil {
```

Comment:
Add documentation on what we ar doing here and why.  Document how we are waiting on the job and what the flow is.